test/fixtures/eval/eval_messages.snapshot

```diff
@@ -11,6 +11,7 @@ SyntaxError: Strict mode code may not include a with statement
 
 
 
+
```

Comment:
nit: did we investigate why this snapshot is showing an extra new line?

Reply:
probably one more function call in the stacktrace, the snapshot transform should remove empty lines